indigo_api/fixtures/drafts.json

```diff
@@ -3,7 +3,7 @@
+      "document_xml": "<akomaNtoso xmlns=\"http://docs.oasis-open.org/legaldocml/ns/akn/3.0\">\n  <act contains=\"originalVersion\">\n    <meta>\n      <identification source=\"\">\n        <FRBRWork>\n          <FRBRthis value=\"/akn/za/act/2014/10/!main\"/>\n          <FRBRuri value=\"/akn/za/act/2014/10\"/>\n          <FRBRalias value=\"This is a title\"/>\n          <FRBRdate date=\"2014-02-12\" name=\"Generation\"/>\n          <FRBRauthor href=\"#council\" as=\"#author\"/>\n          <FRBRcountry value=\"za\"/>\n        </FRBRWork>\n        <FRBRExpression>\n          <FRBRthis value=\"/akn/za/act/2014/10/eng@!main\"/>\n          <FRBRuri value=\"/akn/za/act/2014/12/eng@\"/>\n          <FRBRdate date=\"2014-02-12\" name=\"Generation\"/>\n          <FRBRauthor href=\"#council\" as=\"#author\"/>\n          <FRBRlanguage language=\"eng\"/>\n        </FRBRExpression>\n        <FRBRManifestation>\n          <FRBRthis value=\"/akn/za/act/2014/10/eng@!main\"/>\n          <FRBRuri value=\"/akn/za/act/2014/12/eng@\"/>\n          <FRBRdate date=\"2015-02-04\" name=\"Generation\"/>\n          <FRBRauthor href=\"#council\" as=\"#author\"/>\n        </FRBRManifestation>\n      </identification>\n      <publication date=\"2014-02-12\" name=\"Name of publication\" showAs=\"Name of publication\" number=\"22\"/>\n    </meta>\n    <body>\n      <section eId=\"sec_1\"><num>1.</num>\n        <content>\n          <p>tester</p>\n        </content>\n      </section>\n    </body>\n  </act>\n</akomaNtoso>\n",
```

Comment:
Expression and manifestation URIs missing `/` before `!`

Reply:
thanks, fixed in drafts.json and published.json